### PR TITLE
WRO-6377: Fix security vulnerabilities

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
For preventing malware issues, we need to update node modules that fix severity vulnerabilities.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Update the module to fix 3 moderate severity vulnerabilities
- "webpack-dev-server": "4.9.2"

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Update other dependencies that changed minor version.

### Links
[//]: # (Related issues, references)
WRO-6377

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Comment by hong6316** [comment]
> Why don't we update other dependencies too? I guess it's been 6 months or so.
OK

File: package.json [hunk 1]
```diff
@@ -39,25 +39,25 @@
...
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.7",
+    "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
-    "@testing-library/user-event": "^13.2.0",
-    "@typescript-eslint/eslint-plugin": "^5.5.0",
-    "babel-jest": "^27.5.1",
-    "babel-loader": "^8.2.3",
+    "@testing-library/user-event": "^13.5.0",
+    "@typescript-eslint/eslint-plugin": "^5.30.0",
+    "babel-jest": "^28.1.2",
```
Comment (MikyungKim) [inline package.json:59]: Let's not bump up the major version. It should be aligned with jest.

**Review by MikyungKim — COMMENTED** [review]
Please add a changelog.

File: package.json (hunk 1 above)
Comment (hong6316) [reply package.json:59]: You're right, it may modified accidently.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Updated to the latest `webpack` and `webpack-dev-server` dependency, and changed other dependencies that updated minor version.
```
Comment (MikyungKim) [inline CHANGELOG.md:3]:
```suggestion
* Updated dependencies.
```

File: package.json [hunk 3]
```diff
@@ -39,25 +39,25 @@
...
-    "@enact/dev-utils": "^5.0.0-rc.1",
+    "@babel/plugin-proposal-decorators": "^7.18.6",
+    "@babel/plugin-proposal-export-default-from": "^7.18.6",
+    "@babel/plugin-transform-react-inline-elements": "^7.18.6",
+    "@babel/plugin-transform-runtime": "^7.18.6",
+    "@babel/preset-env": "^7.18.6",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.18.6",
+    "@babel/runtime": "^7.18.6",
+    "@enact/dev-utils": "enactjs/dev-utils#feature/WRO-6377",
```
Comment (MikyungKim) [inline package.json:52]: Please revert this.

File: package.json [hunk 4]
```diff
@@ -68,61 +68,61 @@
...
     "strip-ansi": "^6.0.1",
     "style-loader": "^3.3.1",
     "tar": "^6.1.11",
-    "terser-webpack-plugin": "^5.3.1",
+    "terser-webpack-plugin": "^5.3.3",
     "v8-compile-cache": "^2.3.0",
     "validate-npm-package-name": "^3.0.0",
-    "webpack": "5.64.4",
-    "webpack-dev-server": "4.6.0"
+    "webpack": "^5.73.0",
```
Comment (MikyungKim) [inline package.json:124]:
```suggestion
    "webpack": "5.73.0",
```

File: package.json [hunk 5]
```diff
@@ -68,61 +68,61 @@
...
+    "webpack-dev-server": "^4.9.3"
```
Comment (MikyungKim) [inline package.json:125]:
```suggestion
    "webpack-dev-server": "4.9.3"
```

File: package.json (hunk 3 above)
Comment (MikyungKim) [reply package.json:52]: Let's update the version after releasing dev-utils.

File: package.json (hunk 4 above)
Comment (hong6316) [reply package.json]: OK

File: package.json (hunk 5 above)
Comment (hong6316) [reply package.json]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**